hydroshare/local_settings.py

```diff
@@ -182,3 +182,7 @@
 HYDROSHARE_SHARED_TEMP = '/shared_tmp'
 
 TIME_ZONE = "Etc/UTC"
+
+#used by the mailchimp subscription job in hs_core/tasks.py
+MAILCHIMP_ACTIVE_SUBSCRIBERS = "e210a70864"
+MAILCHIMP_SUBSCRIBERS = "f0c27254e3"
```

Comment:
Will this task run on our development environments? beta? Is there a chance it will accidentally sync too many times or is that not an issue?

Reply:
without the MAILCHIMP_PASSWORD in settings the job will just fail.  dev/beta environments shouldn't have a password.

Reply:
@aphelionz - if you're this is good to go, please complete the review and I'll get it merged.  Where do I add the deployment note to ensure the mailchimp password gets added?